yellowbrick/classifier/rocauc.py

```diff
@@ -395,7 +395,10 @@ def _get_y_scores(self, X):
             try:
                 method = getattr(self.estimator, attr, None)
                 if method:
-                    return method(X)
+                    y_scores = method(X)
+                    if y_scores.ndim == 2 and y_scores.shape[1] == 2:
+                        y_scores = y_scores[:,1]
+                    return y_scores
```

Comment:
Would prefer to use sklearn.utils.multiclass.type_of_target here just as a side note, but we'd also like the user to be able to specify if this is a true binary case or if per_class is actually valid.